### PR TITLE
Track events in Application Insights.

### LINKED ISSUE #471: Add custom events for retrocert app in Azure Application Insights
Application Insights has hourly and minute views for historical data whereas Google Analytics only has daily views, which is less useful for debugging specific failure events. So let's copy the Google Analytics events for the retrocert app as AI events, so that they get logged in Azure and we can view them in more detail.

### PR DESCRIPTION
**Pull request by idealisms**
Also add an event for the language toggle and the back button.

Events are already showing up in App Insights -> [Events](https://portal.azure.com/#@digitalca.onmicrosoft.com/resource/subscriptions/005fa005-b981-47d8-8b39-5b82adf6569c/resourceGroups/rg-cdt-pub-vip-bnscn-w-p-002/providers/microsoft.insights/components/AS-CDT-PUB-VIP-BNSCN-WL-P-002/events)

===

Resolves #471

- [ ] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

Annotation 2020-07-13 132807

### CODE REVIEW

File: src/client/components/LanguageSelector/index.js [hunk 1]
```diff
@@ -2,14 +2,17 @@ import Button from "react-bootstrap/Button";
...
 function LanguageSelector(props) {
   function isEnglish() {
     return i18n.language.startsWith("en");
   }
 
   const toggleLanguage = () => {
-    i18n.changeLanguage(isEnglish() ? "es" : "en");
+    const newLanguage = isEnglish() ? "es" : "en";
+    i18n.changeLanguage(newLanguage);
+    logEvent("Unemployment", "ChangeLanguage", "to-" + newLanguage);
```
Comment (kalvinwang) [inline src/client/components/LanguageSelector/index.js:15]: nice! thank you

File: src/services/azureApplicationInsights.js [hunk 2]
```diff
@@ -43,9 +43,14 @@ const createTelemetryService = () => {
     });
 
     appInsights.loadAppInsights();
+    if (process.env.NODE_ENV === "development") {
+      // Normally metrics are sent as batches. In development mode, send immediately.
```
Comment (kalvinwang) [inline src/services/azureApplicationInsights.js:47]: oh interesting! useful...

Comment (kalvinwang) [reply src/services/azureApplicationInsights.js:47]: is there a super easy way to send immediately for staging as well? no problem if not, just seems potentially useful if we're debugging on staging and watching the metrics in real time

Comment (idealisms) [reply src/services/azureApplicationInsights.js:50]: Sure! I added a domain check so on staging it will be sent immediately.

**Pull request merged**